backend/manager/modules/utils/src/main/resources/languages.properties

```diff
@@ -1,11 +1,12 @@
 #Available languages
 de_DE=Deutsch - Deutschland
 en_US=English - United States
-fr_FR=fran�ais - France
+fr_FR=français - France
 it_IT=Italiano - Italia
-es_ES=espa�ol - Espa�a
+es_ES=español - España
 ja_JP=\u65E5\u672C\u8A9E - \u65E5\u672C
 ko_KR=\uD55C\uAD6D\uC5B4 - \uB300\uD55C\uBBFC\uAD6D
-pt_BR=Portugu�s - Brasil
+pt_BR=Português - Brasil
 zh_CN=\u4E2D\u6587\uFF08\u7B80\u4F53\uFF09 - \u4E2D\u56FD
 cs_CZ=\u010Cesky - \u010Cesko
+ka_GE=ქართული
```

Comment:
This is the only line that should be changed , why are all the former marked as changed ?

Reply:
I believe it is a difference in code page and maybe from github and my vscode:
![vscode](https://user-images.githubusercontent.com/3985964/164502753-a29f5c3c-f7b1-49fb-852f-8bb661665a13.png)
![github](https://user-images.githubusercontent.com/3985964/164502777-33f86032-2374-4fa8-a965-31feb379b0f4.png)

I'll revert the lines.

Reply:
FWIW, here is what the revert diff looks like in my vscode:
![vscode diff](https://user-images.githubusercontent.com/3985964/164503278-ce250317-79a2-489d-a5bb-a84e30c78c1e.png)

Reply:
Very strange.  I had to edit the file outside of vscode to avoid those other changes.

Reply:
```bash
file backend/manager/modules/utils/src/main/resources/languages.properties
backend/manager/modules/utils/src/main/resources/languages.properties: ISO-8859 text
```
the issue is in the file encoding. VSCode uses utf-8 while the file is ISO-8859 encoded.
I would suggest to change the encoding of the file to utf-8 in a separate patch